packages/core/src/lib/scopes.ts

```diff
@@ -0,0 +1,11 @@
+import { RemoteScope } from '../types';
+
+/**
+ *
+ * @returns Returns the window or global object.
+ */
+export const getScope = (): RemoteScope => {
+  return (typeof window === 'undefined'
+    ? global
```

Comment:
Shouldn't this be `global.__remote_scope__`? At least this is what been used in the `integrations/webpack/factory.ts` file

Reply:
Updated